apidef/oas/middleware.go

```diff
@@ -100,11 +100,11 @@ type PluginConfig struct {
 	// Driver configures which custom plugin to be used.
 	// It's value should be set to one of the following:
 	//
-	// - `otto`,
-	// - `python`,
-	// - `lua`,
-	// - `grpc`,
-	// - `goplugin`.
+	// - `otto`
```

Comment:
We have a style convention for this - commas each line, dot trailing line (godoc-like)

Reply:
thanks!!! is there a clear message that I missed? assuming I'm not familiar with tyk repo's ci?